libs/zephyr-agent/src/lib/transformers/ze-build-assets.ts

```diff
@@ -12,7 +12,7 @@ export function zeBuildAssets({
   const buffer = typeof content === 'string' ? Buffer.from(content, 'utf8') : content;
 
   const hash = createHash('sha256')
-    .update(buffer.length ? buffer : Buffer.from(filepath, 'utf8'))
+    .update(buffer)
```

Comment:
if buffer is empty (empty file for example) this will explode

Reply:
I'll revert this one for now.
I am going to get back at this when I do the hash validation though.